### PR TITLE
lt-qcom: switch from stretch to buster

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

**Comment by lumag** [comment]
ping @roxell @fboudra

File: lava_test_plans/projects/lt-qcom/fastboot.jinja2 [hunk 1]
```diff
@@ -21,7 +21,7 @@ protocols:
     name: lxc-target
     template: debian
     distribution: debian
-    release: stretch
+    release: buster
```
Comment (roxell) [inline lava_test_plans/projects/lt-qcom/fastboot.jinja2:24]: why buster and not bullseye ?

Comment (roxell) [reply lava_test_plans/projects/lt-qcom/fastboot.jinja2:24]: Even better would be to move away from lxc and start using docker.

Comment (lumag) [reply lava_test_plans/projects/lt-qcom/fastboot.jinja2:24]: Let's change that to bullseye, agree.
Regarding lxc -> docker, I'd defer this question for now. Maybe @superna9999 can take a look.

**Pull request merged**